packages/discovery/src/discovery/interactive/InteractiveDiffIgnore.ts

```diff
@@ -0,0 +1,196 @@
+    const changesWithDiffs: Change[] = discoveryDiffs.flatMap(
+      (discoveryDiff) => {
+        const contract = discoveryDiff.name
+          ? discoveryDiff.name
+          : discoveryDiff.address.toString()
+
+        if (!discoveryDiff.diff) {
+          return []
+        }
+
+        return discoveryDiff.diff.map((d) => ({
+          contract,
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+          property: d.key!,
+        }))
+      },
+    )
+
+    const rootPropsChanged = getRootPropertiesChanged(changesWithDiffs)
+
+    for (const { property, contract: addressOrName } of rootPropsChanged) {
+      const contract = this.getContract(addressOrName)
+
+      if (!contract) {
+        const msg = `Skipping ${chalk.italic.blue(
+          property,
+        )} - contract not found`
+
+        console.log(msg)
+        continue
+      }
+
+      const prettyContractName = getNameWithColor(contract)
+
+      const isIgnoredInWatchMode = this.dob
+        .getWatchMode(contract)
+        .includes(property)
+
+      if (isIgnoredInWatchMode) {
+        const msg = `Skipping ${chalk.italic.blue(
+          property,
+        )} on ${prettyContractName} - ignored in watch mode`
+        console.log(msg)
+        continue
+      }
+
+      const isIgnoredMethod = this.dob
+        .getIgnoredMethods(contract)
+        .includes(property)
+
+      if (isIgnoredMethod) {
+        const msg = `Skipping ${chalk.italic.blue(
+          property,
+        )} on ${prettyContractName}- ignored method`
+        console.log(msg)
+        continue
+      }
+
+      const isCustomHandler = this.dob.isCustomHandler(contract, property)
+
+      if (isCustomHandler) {
+        const msg = `Skipping ${chalk.italic.blue(
+          property,
+        )} on ${prettyContractName} - custom handler`
+        console.log(msg)
+        continue
+      }
+
+      await this.configureSingleProperty(contract, property)
+    }
+
+    console.log(chalk.green('Flushing overrides...'))
+    await this.dob.flushOverrides()
+    console.log(
+      chalk.green(
+        'Done! Run prettier over the config file. Re-run the discovery if needed.',
+      ),
+    )
+  }
+
+  private async configureSingleProperty(
+    contract: ContractParameters,
+    property: string,
+  ): Promise<void> {
+    const choices = [
+      {
+        name: '👁️  Ignore in watch mode',
+        value: 'watchmode',
+      },
+      {
+        name: '🙅 Ignore method',
+        value: 'method',
+      },
+      {
+        name: '⏩ Skip',
+        value: 'skip',
+      },
+    ] as const
+
+    const name = getNameWithColor(contract)
+
+    const message = `Configure ${chalk.italic.blue(
+      property,
+    )} visibility on ${name}: `
+
+    const choose = await select({
+      message,
+      choices,
+    })
+
+    if (choose === 'watchmode') {
+      const currWatchMode = this.dob.getWatchMode(contract)
+
+      this.dob.setOverride(contract, {
```

Comment:
We should just have a function like `pushOverride`, `appendOverride`, `addOverride` that does just that. The combo of `get-set` smells like Java and is not really practical, same lower for ignored